selftests/unit/test_utils_partition.py

```diff
@@ -68,6 +59,8 @@ def test_basic(self):
         self.disk.unmount()
         self.assertNotIn(self.mountpoint, open("/proc/mounts").read())
 
+    @unittest.skipIf(not process.can_sudo('kill'),
```

Comment:
I'm pretty sorry, but I messed up here.  This line would have to be:

```python
@unittest.skipIf(not process.can_sudo('kill -l')
```

Similar to what is done previously with `mkfs.ext2 -V`.

If you want, I can change that and merge this with that change alone.